### DISCUSSION TITLE
Omarkadash@yahoo.com

### DISCUSSION THREAD

**Discussion by 0792842584** [community/community#19337] [Other Feature Feedback, Questions, & Ideas]
لقد تم اختراق حسابي ولا يمكنني استعادته

**Comment by D7EAD** [comment]
Hi!

If your account has been compromised, I would greatly suggest getting in contact with Github Support. You can do so at [this](https://support.github.com/contact/account) link.

**Comment by 0792842584** [comment]
بتاريخ الأحد، 26 يونيو 2022، جاء من Dread ***@***.***>:
